examples/generate-runtime/pom.xml

```diff
@@ -43,7 +43,7 @@
 
     <dependency>
       <groupId>org.camunda.bpm.springboot</groupId>
-      <artifactId>camunda-bpm-spring-boot-starter-webapp-ee</artifactId>
+      <artifactId>camunda-bpm-spring-boot-starter-webapp</artifactId>
```

Comment:
Temporary workaround: CI is green if we don't pull EE artifacts.

Reply:
If we dont pull EE artifacts do we still need the `camunda-bpm-ee-snapshots` repository?

Reply:
No, but I will leave it for now. We can clean them when we no longer use snapshots at all.